### PR TITLE
V2/add staking pages

### PR DESCRIPTION
**Pull request by dospore**
Re add pool token staking and Balancer pool staking pages

- restyle stake generic table to match Pools table (w.r.t headings and filters)

Follow on tickets
- need to contact someone to know what filters to add to these tables currently the filters are just the old ones

### CODE REVIEW

File: archetypes/Stake/FilterSelects/index.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                                    options={SIDE_OPTIONS}
+                                    onSelect={onSideFilterSelect}
+                                />
+                            </BaseFilters.DropdownContainer>
+                        )}
+                        <BaseFilters.DropdownContainer>
+                            <BaseFilters.Text>Sort</BaseFilters.Text>
+                            <BaseFilters.Dropdown
+                                value={state.sortBy}
+                                options={SORT_BY_OPTIONS}
```
Comment (ScaredIbis) [inline archetypes/Stake/FilterSelects/index.tsx:94]: not sure about these sorting options, they seem a little weird for the staking page (they seem to just be re-used from the pools page)

image

File: context/FarmContext/index.tsx [hunk 2]
```diff
@@ -307,6 +307,7 @@ export const FarmStore: React.FC<
                             farms[farm.address] = farm;
                         }
                     });
+                    console.log('farms', farms);
```
Comment (ScaredIbis) [inline context/FarmContext/index.tsx:310]: can remove

File: utils/farms.ts [hunk 3]
```diff
@@ -10,14 +32,5 @@ export const fetchTokenPrice: (
...
-    const poolInfo = await Pool.Create({
-        ...poolInfo_,
-        provider,
-    });
-
-    return tokenAddresses.map((tokenAddress) => {
-        const isLong: boolean = tokenAddress.toLowerCase() === poolInfo.longToken.address.toLowerCase();
-        return isLong ? poolInfo.getNextLongTokenPrice() : poolInfo.getNextShortTokenPrice();
-    });
+    return tokenAddresses.map((_tokenAddress) => new BigNumber(1));
```
Comment (ScaredIbis) [inline utils/farms.ts:35]: maybe add a TODO here or something so we don't forget

**Review by ScaredIbis — COMMENTED** [review]
mostlty lgtm, couple of points:

the sort filters don't feel right and the collateral tokens you can filter on don't seem to be network-aware (PPUSD is an option on mainnet)

also the new farms are deployed at the following addresses so might as well plug them in:

```
3 LONG ETH/USD
0xbF3E2DbBF663b09EDCe8b774334fc408559846d0

3 SHORT ETH/USD
0xf7824Fce8C155e74E87e9575c7084D0380B55BaF

3 LONG BTC/USD
0xE211c6a34a6b04Df2D5fBCf3E66Fd57b9eD76e0d

3 SHORT BTC/USD
0x50041Fe576cEC7502eA97cE33627856299011Eb1
```

File: utils/farms.ts (hunk 3 above)
Comment (dospore) [reply utils/farms.ts:35]: aight

**Comment by dospore** [comment]
> mostlty lgtm, couple of points:
> 
> * the collateral tokens you can filter on don't seem to be network-aware (PPUSD is an option on mainnet) but can probably address in another PR
> * the new farms are deployed at the following addresses so might as well plug them in:
> 
> ```
> 3 LONG ETH/USD
> 0xbF3E2DbBF663b09EDCe8b774334fc408559846d0
> 
> 3 SHORT ETH/USD
> 0xf7824Fce8C155e74E87e9575c7084D0380B55BaF
> 
> 3 LONG BTC/USD
> 0xE211c6a34a6b04Df2D5fBCf3E66Fd57b9eD76e0d
> 
> 3 SHORT BTC/USD
> 0x50041Fe576cEC7502eA97cE33627856299011Eb1
> ```

haha yeah I actually did this last night as well

**Pull request merged**